www/static/js/docs.js

```diff
@@ -40,7 +40,7 @@ $(document).ready(function () {
 
     // Table of Contents
     $('#page-toc').toc({
-        'selectors':         'h1,h2,h3,h4,h5,h6', // elements to use as headings
+        'selectors':         'h1,h2', // elements to use as headings
```

Comment:
I didn't notice this. We do need this change, but it should be in a separate commit/JIRA. The reason for this @dblotsky is that certain longer pages (like the upcoming single page platforms guide) have a lot of headers that only serve to clutter the toc. Same with reference pages; every attribute does not need a toc entry.